testdata/interface.go

```diff
@@ -30,6 +54,12 @@ func main() {
 	// Try putting a linked list in an interface:
 	// https://github.com/tinygo-org/tinygo/issues/309
 	itf = linkedList{}
+
+	// for i, tc := range interfaceEqualTests {
+	// 	if (tc.lhs != tc.rhs) != tc.shouldFail {
```

Comment:
This code will be easier to read if you avoid the double negative here: `if (tc.lhs == tc.rhs) != tc.shouldBeEqual`.